lib/chef/audit/audit_reporter.rb

```diff
@@ -116,8 +118,14 @@ def post_auditing_data(error = nil)
         Chef::Log.debug("Sending audit report (run-id: #{audit_data.run_id})")
         run_data = audit_data.to_hash
 
-        if error
-          run_data[:error] = "#{error.class.to_s}: #{error.message}\n#{error.backtrace.join("\n")}"
+        if @exception || error
+          errors = [@exception, error].uniq.compact
+          errors_messages = errors.map  do |err|
+            msg = "#{err.class.to_s}: #{err.message}"
+            msg << "\n#{err.backtrace.join("\n")}" if err.backtrace
+            msg
+          end
```

Comment:
If audit phase is enabled and an error is passed to this method, it will either be 1) the audit phase error, 2) the converge phase error or 3) the wrapped error containing both.

If its the audit error or the wrapped error, it will duplicate `@exception`.  If its the converge error, I feel like that should be reported by some converge-oriented reporter.

So I vote no, lets only send the `@exception` error stored earlier from `audit_phase_failed`.